src/rgw/rgw_op.cc

```diff
@@ -5218,7 +5225,9 @@ void RGWDeleteObj::execute(optional_yield y)
       del_op->params.high_precision_time = s->system_request;
       del_op->params.olh_epoch = epoch;
       del_op->params.marker_version_id = version_id;
+      del_op->params.null_verid = null_verid;
 
+      ldpp_dout(this, 1) << "del_op->params.null_verid " << del_op->params.null_verid << dendl;
```

Comment:
at log level 1, this will spam users' rgw logs. can we either remove it or raise the log level to ~20?

Reply:
sorry part of testing. will remove the logging. I don't think we need it.